docs/developer-docs/integrations/sns/tokenomics/preparation.md

```diff
@@ -58,18 +44,11 @@ ledger canister such as the token name, token symbol, and the ledger transaction
       swap tokens are reserved for future swaps (separate from the treasury). 
       Note that future swap are not yet designed, but having the tokens reserved
```

Comment:
I'd cut the part that discusses future swaps.

Reply:
Good call. Removing